code/modules/mapping/map_template.dm

```diff
@@ -157,15 +157,18 @@
 	if((T.y+height) - 1 > world.maxy)
 		return
 
-	var/list/border = block(locate(max(T.x-1, 1), max(T.y-1, 1),  T.z),
-							locate(min(T.x+width+1, world.maxx), min(T.y+height+1, world.maxy), T.z))
+	// Grab the border one larger than the load area, except when at the edge of the map
+	var/turf/bottom_left = locate(max(T.x - 1, 1), max(T.y - 1, 1), T.z)
+	var/turf/top_right = locate(min(T.x + width + 1, world.maxx), min(T.y + height + 1, world.maxy), T.z)
+	var/list/border = block(bottom_left, top_right)
```

Comment:
why this bit?

Reply:
It does the exact same thing, but is more readable

Reply:
the pattern of block(locate(), locate()) is widespread.
should make a macro like `RANGE_TURFS` for it instead

Reply:
This still should be a macro, we do stuff like this often and it hides the intention of the thing better
also it's not a border but that isn't really your fault